maven/lib/dependabot/maven/metadata_finder.rb

```diff
@@ -118,6 +115,12 @@ def dependency_pom_file
         @dependency_pom_file = Nokogiri::XML("")
       end
 
+      def dependency_artifact_id
+        _group_id, artifact_id, = dependency.name.split(":")
```

Comment:
Would be good to also include the classifier here to make it clear we expect the name to have three components in case someone missed this in a future refactor:

```suggestion
        _group_id, artifact_id, _classifier = dependency.name.split(":")
```